BankSDK/GiniBankSDK/Sources/GiniBankSDK/Resources/GiniColors.swift

```diff
@@ -155,3 +156,106 @@ extension UIColor {
         }
     }
 }
+
+extension UIColor {
+    // MARK: Development scheme
+    // TODO: Need to update scheme when designer will finish it
+    public struct GiniColorScheme {
+        struct Bg {
+            var background: GiniColor
+            var surface: GiniColor
+            var tabbar: GiniColor
+            var navbar: GiniColor
+            var inputUnfocused: GiniColor
+            var inputFocused: GiniColor
+            var divider: GiniColor
+            var border: GiniColor
+        }
+
+        struct Button {
+            var surfaceEnabled: GiniColor
+            var textEnabled: GiniColor
+        }
+
+        struct Icons {
+            var surfaceFilled: GiniColor
+            var standardPrimary: GiniColor
+            var standardSecondary: GiniColor
+            var standardTertiary: GiniColor
+            var system: GiniColor
+        }
+
+        struct Text {
+            var system: GiniColor
+            var primary: GiniColor
+            var secondary: GiniColor
+            var tertiary: GiniColor
+            var status: GiniColor
+        }
+
+        struct Chips {
+            var suggestionEnabled: GiniColor
+            var textSuggestionEnabled: GiniColor
+            var assistEnabled: GiniColor
+            var textAssistEnabled: GiniColor
+        }
+
+        struct Toggles {
+            var surfaceFocused: GiniColor
+            var surfaceUnfocused: GiniColor
+            var surfaceDisabled: GiniColor
+            var thumb: GiniColor
+        }
+
+        var bg: Bg
```

Comment:
[suggestion] Please avoid abbreviations.

Reply:
Will be renamed to Background in new schema